media/css/sandstone/sandstone-resp.less

```diff
@@ -902,6 +902,7 @@ nav.menu-bar {
         .border-box();
         width: 100%;
         padding: 15px 10px;
+        height: 53px;
```

Comment:
Rather than introduce a set `height` here, it might be slightly nicer just to change the `padding` directly above to `18px 10px`?